fleet_vehicle_log_fuel/README.rst

```diff
@@ -58,6 +58,10 @@ Contributors
 
 * Miquel Raïch <miquel.raich@forgeflow.com>
 
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Víctor Martínez
```

Comment:
With the previous contribution, we should be here and as co-author, as we rewrite mostly the module.